.github/workflows/comment-integration-failure.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: "Schutzbot comments"
+# This workflow runs when the [integration] workflow reports a failure through
+# an artifact. The [integration] workflow itself doesn't fail if the testing or
+# linting fails, instead it reports a failure through a shared artifact that
+# this workflow reads and reports on. The failure is communicated to the author
+# and reviewers of a PR without causing a failed action to appear.
+#
+# This workflow is run on a workflow_run trigger to have access to secrets.
+#
+# TODO: Actually do what the comment says.
```

Comment:
(nitpick) the comment could be a bit clearer, i.e. something like "//TODO: add integration workflows once this is merge to main ..." or something but the commit message explains the plan so it's a minor thing

Reply:
I really hope the TODO comment doesn't stay around long enough for it to matter, but you're right.  I can change it if you really feel it's important.

Reply:
No, nitpick only :) Let's get it merged